jwst/datamodels/schemas/image.schema.yaml

```diff
@@ -35,5 +35,15 @@ allOf:
       fits_hdu: AREA
       default: 0.0
       datatype: float32
+    wht:
```

Comment:
Just for the purpose of getting correct matching of extensions in fitsdiff comparisons used in the regtests, we need to have the order of the wht and con arrays match what was used in the old DrizProductModel, which had con first and then wht. So if you swap the ordering here we should be able to get complete fitsdiff results for the new vs old files.

Reply:
done